docs/migration.md

```diff
@@ -0,0 +1,86 @@
+# Migration from MMS 0.4
+
+MMS 1.0 is a major release that contains significant architecture improvement based on MMS 0.4.
+
+MMS 1.0 adopted micro-services based architecture, the frontend request handler is separated from backend inference worker. The frontend is a java based web service which provide REST API, and backend is python based worker which execute custom service code (Other language worker support is also planed).
+
+## Table of Content
+
+* [Installation](#installation)
+* [Command line interface](#command-line-interface)
+* [API](#api)
+* [Model archive](#model-archive)
+* [Docker container](#docker-container)
+* [Logging](#logging)
+* [Metrics](#metrics)
+* [Configuration](#configuration)
+    * [SSL](#ssl)
+
+## Installation
+
+MMS 1.0 made following changes for pip installation package:
+
+* **java 8**: java is required for MMS 1.0. You must install java 8 (or later) and make sure java is on available in $PATH environment variable *before* installing MMS. If you have multiple java installed, you can use $JAVA_HOME environment vairable to control which java to use.
```

Comment:
different formatting for java and mxnet?

Reply:
Fixed